src/_includes/banner.html

```diff
@@ -1,5 +1,9 @@
 <div class="banner">
   <p class="banner__text">
-    Dart and Flutter return to Google I/O on May 10! <a href="https://io.google/2023/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_banner&utm_content=" class="no-automatic-external">Register now</a>
+    Dart 3 is out today, along with Google I/O!
```

Comment:
Optional: Maybe increase the desire to read with the two big features?

```suggestion
    Dart 3 is here with records and patterns, along with Google I/O!
```

Reply:
I'm going to change the second part to "alongside Google I/O", just to not say "with" twice. Lmk if that sounds/looks weird!

Reply:
Sure "alongside" sounds good to me!